stock_barcodes_picking_batch_revision/models/stock_move_line.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    is_reviewed = fields.Boolean()
```

Comment:
I think this field has a very common name and can collapse with other developments. It should be prefixed by a distinctful name like `barcodes`:

```suggestion
    barcodes_is_reviewed = fields.Boolean()
```

You can add migration scripts for not losing data.